### ISSUE TITLE
Roll numbers like 14CH3FP18 don't work

### ISSUE THREAD

**Issue by evijit** [Avikalp7/CG-Accumulator#4]

**Comment by evijit** [comment]
@Avikalp7 this is still not addressed. Basically, after you're done with the BTech and Duals you need to scrape the YYDD3FP_ _ and YYDD3EP_ _ Roll numbers.

**Comment by Avikalp7** [comment]
Yup! But one of the problem with the application is its slowness, and since the 3EP and 3FP roll numbers do not end in ordered number for their respective department, I've not been able to find a way to incorporate these without conducting a lengthy search across YYDD3FP/3EP_ _ which will including scraping 40 odd pages for 2/3 results - making the wait time much longer. One solution is hard-coding but that doesn't seem good. Please let me know if there could be a more efficient way. Pull request would be great :D !

**Comment by evijit** [comment]
Have you thought about caching the data periodically somewhere? This is especially needed when the site goes down due to the Senate approval thing. If the results are displayed from the database and the scraper works as a separate unit, the main application will become significantly faster.
